enclave_manager/avalon_enclave_manager/work_order_kv_delegate.py

```diff
@@ -102,9 +120,9 @@ def cleanup_work_orders(self, wo_ids=None):
             else:
                 logger.info("No response found for the workorder %s; " +
                             "hence placing the workorder request " +
-                            "back in wo-scheduled", wo)
-                self._kv_helper.set("wo-scheduled", wo,
-                                    WorkOrderStatus.SCHEDULED.name)
+                            "back in wo-worker-pending", wo)
+                self._kv_helper.csv_prepend("wo-worker-pending",
```

Comment:
What does "pending" state of work order mean ?

Reply:
"pending" implies submitted but not picked up for processing yet.